modules/ROOT/pages/administration/access-control/dbms-administration.adoc

```diff
@@ -71,13 +71,13 @@ This section explains how to use Cypher to manage Neo4j DBMS administrative priv
 
 All DBMS privileges are relevant system-wide.
 Like user management, they do not belong to one specific database or graph.
-For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/neo4j-databases-graphs.adoc[].
+For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/cypher_neo4j.adoc[].
 
 image::privileges_grant_and_deny_syntax_dbms_privileges.svg[title="Syntax of GRANT and DENY DBMS Privileges"]
 
 image::privileges_hierarchy_dbms.svg[title="DBMS privileges hierarchy"]
```

Comment:
These images don't seem to render :(

Reply:
Good spot - they don't render in the published docs either :/ will look into it!